sympy/stats/crv_types.py

```diff
@@ -1840,7 +1840,7 @@ class LogNormalDistribution(SingleContinuousDistribution):
     @staticmethod
     def check(mu, std):
         _value_check(mu.is_real, "mu, location must be real.")
-        _value_check(std > 0, "s, scale must be positive")
+        _value_check(std >= 0, "s, stdandard deviation must be positive")
```

Comment:
stdandard -> standard

Can it really be 0?

Reply:
You are right, I don't know what I was thinking while making this change. That line doesn't need any change. Correcting.